### PR TITLE
Add reference to service workers, page visibility

### PR DESCRIPTION
**Pull request by QingAn**
***
<a href="https://pr-preview.s3.amazonaws.com/QingAn/miniapp-lifecycle/pull/18.html" title="Last updated on Sep 24, 2021, 9:50 AM UTC (123be52)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-lifecycle/18/13c17b5...QingAn:123be52.html" title="Last updated on Sep 24, 2021, 9:50 AM UTC (123be52)">Diff</a>

***
<a href="https://pr-preview.s3.amazonaws.com/QingAn/miniapp-lifecycle/pull/18.html" title="Last updated on Feb 18, 2022, 11:07 AM UTC (62e89c9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/miniapp-lifecycle/18/13c17b5...QingAn:62e89c9.html" title="Last updated on Feb 18, 2022, 11:07 AM UTC (62e89c9)">Diff</a>

### CODE REVIEW

**Comment by xfq** [comment]
Note that we haven't added the mapping to the spec yet. I suggest that we add this sentence after we add the mapping to the spec.

File: index.html [hunk 1]
```diff
@@ -168,6 +168,10 @@ <h2>MiniApp Global Application Lifecycle States</h2>
           Lifecycle state for MiniApp [=Document/in error=]. It is triggered once the MiniApp is confronted with script error.
         </dd>
       </dl>
+            
+         <p>
+            The mapping to existing Web specifications such as <a href="https://www.w3.org/TR/service-workers/">Service Workers</a> is listed in <a href="https://github.com/w3c/miniapp-lifecycle/blob/main/docs/explainer.md#comparison-with-some-related-work-in-w3c-such-as-service-worker-page-visibility-and-page-lifecycle">MiniApp Lifecycle Explainer</a>.
```
Comment (xfq) [inline index.html:173]: As I said in https://github.com/w3c/miniapp-lifecycle/issues/15#issuecomment-925673970 , the "mapping" is to demonstrate how to polyfill the behavior of MiniApp Lifecycle using existing Web APIs. The "mapping" in the explainer may be considered too rough.

But at the same time, our [charter](https://www.w3.org/2021/01/miniapps-wg-charter.html#rec-track) says "**whenever possible**, the specification should provide a mapping to existing Web specifications such as [Service Workers](https://www.w3.org/TR/service-workers/) and [Page Visibility](https://w3c.github.io/page-visibility/)", so it is not a MUST.

Comment (QingAn) [reply index.html:173]: The "mapping" comment from TAG is about the explainer (https://github.com/w3ctag/design-reviews/issues/523#issuecomment-928975772), which has been somewhat accepted by TAG (https://github.com/w3ctag/design-reviews/issues/523#issuecomment-1032411658). Therefore, besides updating the explainer, I also add some reference in Lifecycle spec.

Or, we can simply choose to only update the explainer, not the spec. What do you think?

Comment (xfq) [reply index.html:173]: Works for me.

Comment (QingAn) [reply index.html:173]: Consensus to merge in MiniApp WG meeting 2022-02-24

**Pull request merged**